From cac882449b78da053140d5e95f59ae1882e50459 Mon Sep 17 00:00:00 2001
From: "Yichun Zhang (agentzh)" <yichun@openresty.com>
Date: Sat, 9 Mar 2019 11:35:17 -0800
Subject: [PATCH] tests: fixed the tests using iscribblet.org which would time
 out now.

---
 t/091-coroutine.t | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/t/091-coroutine.t b/t/091-coroutine.t
index bceb5124..18cd838b 100644
--- a/t/091-coroutine.t
+++ b/t/091-coroutine.t
@@ -178,7 +178,7 @@ cc3: 2
 
             local urls = {
                 "agentzh.org",
-                "iscribblet.org",
+                "openresty.com",
                 "openresty.org"
             }
 
@@ -204,7 +204,7 @@ cc3: 2
 GET /lua
 --- response_body
 successfully connected to: agentzh.org
-successfully connected to: iscribblet.org
+successfully connected to: openresty.com
 successfully connected to: openresty.org
 *** All Done ***
 --- no_error_log
@@ -377,7 +377,7 @@ GET /lua
 
             local urls = {
                 "agentzh.org",
-                "iscribblet.org",
+                "openresty.com",
                 "openresty.org"
             }
 
@@ -398,7 +398,7 @@ GET /lua
 GET /lua
 --- response_body
 successfully connected to: agentzh.org
-successfully connected to: iscribblet.org
+successfully connected to: openresty.com
 successfully connected to: openresty.org
 *** All Done ***
 --- no_error_log
-- 
2.20.1

